### ISSUE TITLE
Document title has suspicious name

### ISSUE THREAD

**Issue by anna-guo-level-access** [anna-guo-level-access/a#30]
**Rule description**  
The first document title element located does not have a meaningful value.

**Tool name**  
Access Engine

**Severity**  
High

**WCAG Success criteria**  
[2.4.2 Page Titled - Level A](https://www.w3.org/WAI/WCAG22/Understanding/page-titled)

**Why it matters**  
Without a meaningful document/page title, users with cognitive disabilities and those who are blind or low vision may have difficulty determining the topic or purpose of the page.

**What to do**  
Provide a meaningful <code>title</code> element that describes the topic or purpose of the page while avoiding containing extra information that does not add to understanding. 

The title of the page needs to be in the primary language of the page and the value for the <code>title</code> element needs to be updated if the page context changes. 

When the title contains both specific and more general information front load the most specific information at the beginning of the page title.

**Task in the Level Access Platform**  
https://tenantb.localhost-dashboard.essentialaccessibility.com/my-tasks/6a3c613b43450165564cc432/6a47cf8eec978c05f4013f50

**Comment by anna-guo-level-access** [comment]
<a href="https://tenantb.localhost-dashboard.essentialaccessibility.com/tasks/6a47cf8eec978c05f4013f50/attachments/6a47cf8eec978c05f4013f4e/download" target="_blank" rel="noopener noreferrer">Linked Findings.csv</a>

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access
